scripts/Synapse/convertParquetMPI.ipynb

```diff
@@ -136,6 +142,17 @@
         "            'bundle': fhir_bundle,\n",
         "            'external_person_id': iris_id\n",
         "        }\n",
+        "\n",
+        "        # Store a snapshot of the FHIR bundle before it hits standardization\n",
+        "        pre_write_data = {\n",
+        "            'blob': {\"FhirResource\": fhir_bundle},\n",
+        "            'cloud_provider': 'azure',\n",
+        "            'bucket_name': 'bundle-diffs',\n",
```

Comment:
Two thoughts here:

1. This container doesn't currently exist and this service isn't smart enough to create it for us, so well need to add terraform coverage for this. [Here](https://github.com/CDCgov/phdi-azure/blob/main/terraform/modules/shared/main.tf#L22) is an example of where we do this for the `source-data` container.
2. We aren't really storing the diffs so personally I don't think "bundle-diffs" is a good name. What about "bundle-snapshots"?